extension/chrome/settings/modules/my_key.ts

```diff
@@ -50,27 +51,27 @@ Catch.try(async () => {
   $('.show_when_showing_public').css('display', '');
   $('.show_when_showing_private').css('display', 'none');
 
-  $('.action_download_pubkey').click(Ui.event.prevent('double', () => {
-    Browser.saveToDownloads(Att.keyinfoAsPubkeyAtt(primaryKi), Catch.browser().name === 'firefox' ? $('body') : undefined);
+  $('.action_download_pubkey, .action_download_prv').click(Ui.event.prevent('double', (e) => {
+    const keyStr = e.className.includes('pubkey') ? primaryKi.public : primaryKi.private;
```

Comment:
I really appreciate that you made it into one handler, but I'll refactor it as two handlers. Sorry - this is a rare case that's very security sensitive. I've caused a vulnerability before by wrongly using private key instead of pubkey before, so I'd rather have this separate, to make it harder to introduce a vulnerability when someone edits it in the future.